src/linear_algebra/finite_dimensional.lean

```diff
@@ -1759,6 +1749,42 @@ theorem subalgebra.dim_eq_one_iff {S : subalgebra F E} : module.rank F S = 1 ↔
 theorem subalgebra.finrank_eq_one_iff {S : subalgebra F E} : finrank F S = 1 ↔ S = ⊥ :=
 ⟨subalgebra.eq_bot_of_finrank_one, subalgebra.finrank_eq_one_of_eq_bot⟩
 
+lemma subalgebra.bot_eq_top_iff_dim_eq_one :
+  module.rank F E = 1 ↔ (⊥ : subalgebra F E) = ⊤ :=
```

Comment:
This lemma is either stated or named backwards

Reply:
Good point! I decided to keep the names and change the direction of the iff to have the simplest statements on the RHS.